.github/workflows/publish-ocr-extraction-taskrunner.yml

```
@@ -0,0 +1,109 @@
+---
+# Automatically build and publish the ocr-extraction-taskrunner to the container registry.
+# This workflow is triggered on changes to the ocr-extraction-taskrunner directory.
+
+name: Build and Publish ocr-extraction-taskrunner
+
+# yamllint disable-line rule:truthy
+on:
+  push:
```

```suggestion
on:
  workflow_dispatch:
  push:
```

This allow us to re-run the action without pushing code change in the two paths below. Useful in some situation. I added this into `mira-taskrunner` but not all yet.